Shell/Options.cpp

```diff
@@ -2046,6 +2066,12 @@ void Options::init()
     _splitAtActivation.onlyUsefulWith(_splitting.is(equal(true)));
     _splitAtActivation.tag(OptionTag::AVATAR);
 
+    _cleaveNonsplittables = BoolOptionValue("cleave_nonsplittables","cn",false);
+    _cleaveNonsplittables.description="Tentatively propose single-literal component strengthenings. Sometimes useful for bringing about finite saturations.";
```

Comment:
Weirdly seems to be in unsat schedules too. I guess these aren't just noise? I really don't know what I unleashed.

Reply:
Ahah - yes, fooled by EPR.